client/dashboard/index.js

```diff
@@ -9,6 +9,7 @@ import { Component, Fragment } from '@wordpress/element';
  * Internal dependencies
  */
 import './style.scss';
+import ChartExample from 'components/chart/example-hour';
```

Comment:
I had some trouble understanding that `WidgetCharts`, `ChartExample` and `example-hour` referred to the same component. :smile: I understand that component is just a placeholder for now, right? Otherwise, would it be possible to unify the naming, or is there any reason against it?